client/components/sites-dropdown/test/index.js

```diff
@@ -32,18 +32,30 @@ describe( 'index', function() {
 	describe( 'component rendering', function() {
 		it( 'should render a dropdown component initially closed', function() {
 			const sitesDropdown = shallow( <SitesDropdown /> );
-			expect( sitesDropdown.hasClass( 'sites-dropdown' ) ).to.be.true;
-			expect( sitesDropdown.hasClass( 'is-open' ) ).to.be.false;
+			expect( sitesDropdown.hasClass( 'sites-dropdown' ) ).to.equal( true );
+			expect( sitesDropdown.hasClass( 'is-open' ) ).to.equal( false );
 		} );
 
-		it( 'should toggle the dropdown, when it is clicked', function() {
+		it( 'with multiple sites, should toggle the dropdown when it is clicked', function() {
```

Comment:
I like those tests, I think it is enough to make sure we provide different visual state depending on the number of sites.